lib/active_model/serializer/concerns/configuration.rb

```diff
@@ -31,6 +31,28 @@ def config.array_serializer
         # ref: http://jsonapi.org/format/#document-top-level
         config.jsonapi_include_toplevel_object = false
 
+        # example output
+        # => [
+        # "::Api::V1::ResNamespace::ResourceSerializer",
+        # "::Api::ResNamespace::ResourceSerializer",
+        # "::ResNamespace::ResourceSerializer" ,
+        # "::ResourceSerializer"]
+        config.serializer_lookup = proc { |controller_name|
+          lookups = []
+          current_resource = controller_name.gsub(/Controller\z/, '')
```

Comment:
controller_name isn't correct. It's more lookup_namespace and then you want to `lookup_namespace.remove(/Controller\z/)` because you know someone might pass in a controller?